File: FakeNews/FakeNews.py
import pandas as pd

FakeNews = pd.read_excel('C:/User/Hanna B/Documents/python_course/Practice HandsOn L1/FakeNews.xlsx')
FakeNews.head()

#Add a column labeled StoryType and fill it with Fake

FakeNews['StoryType'] = "Fake"
FakeNews.head()

#Remove the when column

FakeNews.drop(['when'], axis=1, inplace=True)
FakeNews.head()




#Separate the URL column into Website and Domain

FakeNews1 = FakeNews['url'].str.split('_', expand=True).rename(columns = lambda x: "URL"+str(x+1))
FakeNews1.head()

FakeNews2 = pd.concat([FakeNews, FakeNews1], axis=1)
FakeNews2.head()

FakeNews2.drop(['url'], axis=1, inplace=True)
FakeNews2.head()



#Put the domain column back together:

FakeNews2['url'] = FakeNews2["URL1"] + "_" + FakeNews2["URL2"].map(str)
FakeNews2.head()

FakeNews2.drop(['URL1', 'URL2'], axis=1, inplace=True)
FakeNews2.head()

#Keep ONLY the first ten rows of the data


FakeNews3 = FakeNews2[:10]
FakeNews3

File: FakeNews/FakeNews.R
library("tidyr")


#Add column labeled StoryType and fill it with Fake:

  FakeNews$StoryType = "Fake"

#Remove the "when" column:
  
  FakeNews1 <- FakeNews[, 2:4]
  
#Separate the URL column 
#To see the website in one column and the domain in another:
  
  FakeNews2 <- separate(FakeNews1, url, c("Website", "Domain"), sep="_")
  
  
#Put the domain column back together:
  
  FakeNews3 <- unite(FakeNews2, FullSiteName, Website, Domain, sep = "_")
  
  
#Keep ONLY the first ten rows of data:
  
  FakeNews4 <- FakeNews3[1:10,]
  
  
  
  
